### PR TITLE
[SYCL] Add CODEOWNERS for Matrix tests

### PR DESCRIPTION
**Pull request by aelovikov-intel**

### CODE REVIEW

**Review by stdale-intel — APPROVED** [review]

**Review by dkhaldi — COMMENTED** [review]
Can we add @yubingex007-a11y as well to the matrix list?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -100,3 +100,6 @@ sycl/source/detail/jit_compiler.cpp @victor-eds @Naghasan @sommerlukas
 sycl/source/detail/jit_device_binaries.hpp @victor-eds @Naghasan @sommerlukas
 sycl/source/detail/jit_device_binaries.cpp @victor-eds @Naghasan @sommerlukas
 sycl/test-e2e/KernelFusion @victor-eds @Naghasan @sommerlukas
+
+# Matrix tests
+sycl/test-e2e/Matrix @dkhaldi @YuriPlyakhin
```
Comment (aelovikov-intel) [inline .github/CODEOWNERS:105]:
```suggestion
sycl/test-e2e/Matrix @dkhaldi @YuriPlyakhin @yubingex007-a11y
```

**Comment by aelovikov-intel** [comment]
> Can we add @yubingex007-a11y as well to the matrix list?

Done

**Review by dkhaldi — APPROVED** [review]
LGTM

**Comment by aelovikov-intel** [comment]
@dkhaldi @YuriPlyakhin @yubingex007-a11y, we expect all of you to approve this PR before it gets merged in.

**Review by YuriPlyakhin — APPROVED** [review]

**Review by yubingex007-a11y — APPROVED** [review]

**Review by bader — APPROVED** [review]

**Pull request merged**